src/ai/backend/manager/models/image.py

```diff
@@ -1022,6 +1022,17 @@ async def _get_user_accessible_project_scopes(
 
         return [ProjectScope(project_id=group_id) for group_id in group_ids]
 
+    async def _get_domain_accessible_project_scopes(
+        self,
+        ctx: ClientContext,
+        scope: DomainScope,
+    ) -> list[ProjectScope]:
+        from .group import GroupRow
+
+        stmt = sa.select(GroupRow.id).where(GroupRow.domain_name == scope.domain_name)
+        project_ids = await self.db_session.scalars(stmt)
+        return [ProjectScope(project_id=proj_id) for proj_id in project_ids]
+
```

Comment:
There are multiple functions now, but since RBAC refactoring is scheduled, I will apply it as is for now.